assets/_scss/components/_footer.scss

```diff
@@ -0,0 +1,298 @@
+.usa-footer {
+
+  button {
+    width: 100%;
+  
+    @media (min-width: $tablet-up) {
+      width: auto;
+    }
+  }
+
+  .usa-unstyled-list {
+    display: block;
+  }
+
+  .usa-footer-primary-link {
+    border-top: 1px solid $color-base-link;  
+    color: $color-base-link;
+    display: block;
+    font-weight: $font-bold;
+    margin-top: 0;
+    padding: {
+      bottom: 2rem;
+      top: 2rem;
+    }
+    text-transform: uppercase;
+
+    @media (min-width: $tablet-up) {
+      border-top: none;
+      font-size: 1.9rem;
+      text-transform: none;
+    }  
+  }  
+}
+
+.usa-footer a {
+  font-weight: normal;
+}
+
+.usa-footer-return-to-top {
+  padding: {
+    bottom: 2rem;    
+    top: 2rem;
+  }
+}
+
+.usa-footer-primary-section {
+  background-color: $color-grey-100;
+
+  a, 
+  p, 
```

Comment:
These rules `a` and `p` here scare me a little. We're saying all `<a>` and `<p>` tags should having padding on them. I could see this potentially being restricting if somebody wanted to customize the footer to their own liking.

Then again, I don't know how the users will be using the pattern lib, so maybe they won't be modifying it like I say.